content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -158,6 +158,11 @@ tables to provide per-instance subnets to each host (which is limited to 50-100
 entries per VPC). In short, cni-ipvlan-vpc-k8s significantly reduces the
 network complexity required to deploy Kubernetes at scale within AWS.
 
+### Coil
+
+[Coil](https://github.com/cybozu-go/coil) is a CNI plugin designed for easy-integration and flexible egress networking.
+Coil provides bare metal performance and allows you to define arbitrary egress NAT gateways for external networks.
```

Comment:
```suggestion
Coil operates with a low overhead compared to bare metal, and allows you to define arbitrary egress
NAT gateways for external networks.
```
I'm inferring that Coil has _some_ overhead but that it's really low - does that seem fair?

Reply:
Yes, thank you.  I will update.